src/components/Swap/Toolbar/Caption.tsx

```diff
@@ -151,59 +174,38 @@ export function Trade({
   gasUseEstimateUSD,
   expanded,
   loading,
+  warning,
 }: TradeProps & TradeTooltip & ExpandProps) {
+  const widgetWidth = useWidgetWidth()
+  const shouldHideUSD = widgetWidth < 360 && warning && !expanded
```

Comment:
this behavior is based on a conversation with Fred  - at narrow widget width, we need to shorten the Trade Price string length so the new button fits

Reply:
Is there a constant that can be used instead of `360`?
You should include a comment above this line explaining this behavior. A good rule of thumb is that if you leave a PR comment, it should probably be noted in the code.

Reply:
updated to use a share constant that already exists